.infrastructure/pv.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: todoapp-pv
+spec:
+  persistentVolumeReclaimPolicy: Delete
+  storageClassName: standard
+  accessModes:
+    - ReadWriteMany
+  capacity:
+    storage: 1Gi
+  hostPath:
+    path: /data/
```

Comment:
Using `hostPath` for a PersistentVolume is generally not recommended for production environments as it ties the volume to a specific node. Consider using a more flexible storage solution like NFS, cloud storage, or a dynamic provisioner that is compatible with your Kubernetes cluster.